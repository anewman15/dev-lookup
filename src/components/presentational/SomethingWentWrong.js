import common from '../../styles/commonStyles.module.css';

const SomethingWentWrong = () => (
  <div className={`${common.container} ${common.flexCenter}`}>
    <p className={`${common.boxRed}`}>Something&apos;s WRONG. Maybe Your Internet?</p>
  </div>
);

export default SomethingWentWrong;
