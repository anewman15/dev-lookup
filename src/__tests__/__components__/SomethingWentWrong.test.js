import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import SomethingWentWrong from '../../components/presentational/SomethingWentWrong';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('SomethingWentWrong', () => {
  it('renders the something is wrong message', () => {
    const tree = renderer
      .create(<SomethingWentWrong />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
