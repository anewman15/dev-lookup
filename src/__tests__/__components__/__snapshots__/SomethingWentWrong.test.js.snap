// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomethingWentWrong renders the something is wrong message 1`] = `
<div
  className="container flexCenter"
>
  <p
    className="boxRed"
  >
    Something's WRONG. Maybe Your Internet?
  </p>
</div>
`;
